### PR TITLE
Remove Kickoff traces

### LINKED ISSUE #311: Cleanup of launch event
As mentioned by @ClaireWyatt in [Slack](https://ukrse.slack.com/archives/GTATB7ATS/p1599122127033300) we need to cleanup some things regarding the kick-off event including:

* [x] removing button in the top banner on front page
* [x] changing text on kick off page to past tense
* [x] removing join and register buttons
* [x] ~~adding materials, if available to indico~~
* [x] integrating the recoding

### PR DESCRIPTION
**Pull request by sdruskat**
This PR is as per https://ukrse.slack.com/archives/GTATB7ATS/p1599122127033300 removing unneeded traces of the kickoff session from the website.

Fixes #311

### CODE REVIEW

**Comment by eileen-kuehn** [comment]
Now we got two solutions... I will try to revert my changes and push into your PR now.

File: _posts/programme/2020-08-19-kickoff.md [hunk 1]
```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
 meeting_url: https://zoom.us/j/95982897830
 last_modified_at: 2020-09-02
 ---
-To kick our series off, we’re delighted to announce that the SORSE Launch
-Event will take place 13:00-15:00 UTC on Wednesday 2nd September. We’re
-pleased to be welcoming two keynote speakers at this event:
+To kick off our series off, we have run the SORSE Launch
```
Comment (Chilipp) [inline _posts/programme/2020-08-19-kickoff.md:17]:
```suggestion
To kick off our series, we ran the SORSE Launch
```

**Review by Chilipp — APPROVED** [review]

File: _posts/programme/2020-08-19-kickoff.md [hunk 2]
```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
 meeting_url: https://zoom.us/j/95982897830
 last_modified_at: 2020-09-02
```
Comment (Chilipp) [inline _posts/programme/2020-08-19-kickoff.md:15]:
```suggestion
last_modified_at: 2020-09-04
```

File: _posts/programme/2020-08-19-kickoff.md [hunk 3]
```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
...
-    {% include add-to-calendar-button.html %}
-</div>
-
-The session will begin with a short welcome and introduction to SORSE followed
-by two 30 minute keynote talks. Dr Mariann Hardey will speak on the subject of
-women in tech and Dr Kari Jordan will speak about the Carpentries (both
-recorded). Further details and abstracts can be seen below. The talks will be
+The session began with a short welcome and introduction to SORSE followed
+by two 30 minute keynote talks. Dr Mariann Hardey spoke on the subject of
+women in tech and Dr Kari Jordan spoke about the Carpentries. The talks were
```
Comment (eileen-kuehn) [inline _posts/programme/2020-08-19-kickoff.md:26]:
```suggestion
by two 30 minute keynote talks. Dr Kari Jordan spoke about the Carpentries 
and Dr Mariann Hardey spoke on the subject of women in tech. The talks were
```

File: _posts/programme/2020-08-19-kickoff.md [hunk 4]
```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
...
 followed by time for questions and then an opportunity for further discussion
 and networking in breakout rooms.
 
-## Joining instructions
-The event will take place as a Zoom Webinar and will switch to a regular Zoom session for discussion and networking in breakout groups. Connection details
-will be sent to all registered participants in advance of the event.
-
-If you haven't registered, you can still join by clicking the _Join_ button
-above.
+<i class="fab fa-fw fa-youtube"></i> A recording of this event is [available on YouTube](https://www.youtube.com/watch?v=2HOznlOCd1w).
```
Comment (eileen-kuehn) [inline _posts/programme/2020-08-19-kickoff.md:30]: What about also adding the video directly into the site? Should be easy through the [youtube helper](https://mmistakes.github.io/minimal-mistakes/docs/helpers/#youtube) provided by our theme.

**Review by eileen-kuehn — COMMENTED** [review]
Looks pretty good!
I left two minor comments. If we want to integrate the video directly into the site, I can go for it, if you like me to.

**Comment by Chilipp** [comment]
:+1: for embedding :smiley:

File: _posts/programme/2020-08-19-kickoff.md (hunk 1 above)
Comment (sdruskat) [reply _posts/programme/2020-08-19-kickoff.md:17]: Ah, nice catch, the duplicate off. Thanks.

I prefer perfect in this case as it it something that reverberates into the present, but either way is fine.

File: _posts/programme/2020-08-19-kickoff.md (hunk 3 above)
Comment (sdruskat) [reply _posts/programme/2020-08-19-kickoff.md:26]: Have thought about the order while working on this, but was pressed for time, so didn't change. Nice one!

File: _posts/programme/2020-08-19-kickoff.md (hunk 4 above)
Comment (sdruskat) [reply _posts/programme/2020-08-19-kickoff.md:30]: Sure!

File: _posts/programme/2020-08-19-kickoff.md (hunk 1 above)
Comment (Chilipp) [reply _posts/programme/2020-08-19-kickoff.md]: As you like. I just used simple past because you used it down below, too.

Comment (sdruskat) [reply _posts/programme/2020-08-19-kickoff.md]: Nah, it's fine :).

**Comment by sdruskat** [comment]
Good to go from my side, please merge @eileen-kuehn / @Chilipp. I'm alwasy hesitant to merge my own PRs ;).

**Comment by Chilipp** [comment]
For me, everything's fine, too!

https://1100-267395254-gh.circle-artifacts.com/0/SORSE/programme/kickoff/index.html

Thanks a lot @sdruskat and @eileen-kuehn :smiley:

@eileen-kuehn please merge if you are done with it

**Comment by eileen-kuehn** [comment]
Ok, happy to merge when checks are done 👍 
I will open another PR then that introduces a widget for the videos. I am currently working on displaying those for `event_001.md` and `event_002.md` with appropriate timings to directly jump to the presentations.

**Pull request merged**